.github/workflows/check.yaml

```diff
@@ -31,6 +31,7 @@ jobs:
       fail-fast: false
     with:
       command: "TEST_JUJU3=1 make functional"  # using TEST_JUJU3 due https://github.com/openstack-charmers/zaza/commit/af7eea953dd5d74d3d074fe67b5765dca3911ca6
+      runs-on: "['self-hosted', 'xlarge', 'jammy', 'x64']"
```

Comment:
This is passed to our re-usable workflow, where we are parsing as json, so it will become a list. We cannot define input argument as list/array/... so we must use string and parsed it with json.
See: https://github.com/orgs/community/discussions/11692

Reply:
It was not added before and since default value is "['self-hosted']" it was picked up by arm64 runner -> CI failed.

Reply:
We are using multiple versions / branches and you can find this in v3 [here](https://github.com/canonical/bootstack-actions/blob/v3/.github/workflows/func.yaml#L54-L58). Agreed, we are lacking of any documentation for those actions from main, v2 and v3.